test/integration/core/core-commands.bats

```diff
@@ -71,6 +67,11 @@ function setup() {
   [[ ${output} == *"not running. Please start this with"* ]]
 }
 
+@test "$DRIVER: machine should now allow upgrade when stopped" {
```

Comment:
Should this be `machine should not allow upgrade when stopped` -- "not" instead of "now"?